gradio/components/multimodal_textbox.py

```diff
@@ -171,15 +170,15 @@ def preprocess(self, payload: MultimodalData | None) -> MultimodalValue | None:
             "files": [f.path for f in payload.files],
         }
 
-    def postprocess(self, value: MultimodalValue | str | None) -> MultimodalData:
+    def postprocess(self, value: MultimodalValue | str | None) -> MultimodalData | None:
         """
         Parameters:
             value: Expects a {dict} with "text" and "files", both optional. The files array is a list of file paths or URLs.
         Returns:
             The value to display in the multimodal textbox. Files information as a list of FileData objects.
         """
         if value is None:
-            return MultimodalData(text="", files=[])
+            return None
```

Comment:
This is fine since we coerce in the frontend anyways:
https://github.com/gradio-app/gradio/blob/a966e9f753af25eb9d813dfdbce39be6f3014d82/js/multimodaltextbox/shared/MultimodalTextbox.svelte#L69